### DISCUSSION TITLE
I can't access my existing account on github

### DISCUSSION THREAD

**Discussion by tvnr** [community/community#21994] [New to GitHub]
<p>My account is currently available for viewing on github, but when I tried to log in to my account, I was greeted with a page stating that my account had been closed. although my account can be viewed, I can’t access my account, please help. My inaccessible account (<a href="http://github.com/30l" class="inline-onebox" rel="noopener nofollow ugc">30l · GitHub</a>)<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/d/4da6caae32705e27a739963b26215dcb498887b5.png" data-download-href="/uploads/short-url/b4W77r37wbuLoGJ3JdM2MikaoUl.png?dl=1" title="unknown-1" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/94189777/181068096-80345279-2460-4afa-9849-2435d19b8d45.png" alt="unknown-1" data-base62-sha1="b4W77r37wbuLoGJ3JdM2MikaoUl" width="491" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4da6caae32705e27a739963b26215dcb498887b5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">unknown-1</span><span class="informations">533×542 18.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/d/fde7e2101579cdf9a24ac1a08bc9de4a60c47ace.png" data-download-href="/uploads/short-url/Ae9BE0W8BCfIE5ohhdfHPqI28RE.png?dl=1" title="unknown (2)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/94189777/181068097-e0058a92-4c4c-4b97-be16-6c823fda4783.png" alt="unknown (2)" data-base62-sha1="Ae9BE0W8BCfIE5ohhdfHPqI28RE" width="690" height="375" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/d/fde7e2101579cdf9a24ac1a08bc9de4a60c47ace_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">unknown (2)</span><span class="informations">758×413 9.33 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>As it says there: Contact support. GitHub staff can’t discuss private account details on this public forum.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com/contact/account">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/5814919/181068098-f02590ad-0ada-4a17-9792-b1ea653a845d.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com/contact/account" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com/contact/account" target="_blank" rel="noopener">Accounts - GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
